wsd/DocumentBroker.cpp

```diff
@@ -4567,6 +4570,30 @@ void DocumentBroker::handleMediaRequest(std::string range,
     }
 }
 
+bool DocumentBroker::requestTileRendering(TileDesc& tile, bool forceKeyframe,
+                                          const std::chrono::steady_clock::time_point &now,
+                                          std::vector<TileDesc>& tilesNeedsRendering,
+                                          const std::shared_ptr<ClientSession>& session)
+{
+    bool allSamePartAndSize = true;
+    if (!tileCache().hasTileBeingRendered(tile, &now) || // There is no in progress rendering of the given tile
+        tileCache().getTileBeingRenderedVersion(tile) < tile.getVersion()) // We need a newer version
+    {
+        tile.setVersion(++_tileVersion);
+        if (forceKeyframe)
+        {
+            LOG_TRC("Forcing keyframe for tile was oldwid " << tile.getOldWireId());
+            tile.setOldWireId(0);
```

Comment:
forceKeyframe() is just { setOldWireId(0); }, so if we replace setOldWireId(0) with that alias, then it becomes more clear that in the call from DocumentBroker::handleTileCombinedRequest that the forceKeyFrame argument to requestTileRendering doesn't really matter, that tile is already forceKeyframe()-ed by handleTileCombinedRequest if it passes forceKeyframe true to here

Reply:
yes =)